### PR TITLE
#900 upgraded netbout-client dependencies

### PR DESCRIPTION
**Pull request by exper0**
#900 upgraded netbout-client dependencies:
jcabi-http  1.10.4 -> 1.14.1    
h2  1.4.186 -> 1.4.190 
jersey-client   1.18.1 -> 1.19

### CODE REVIEW

File: netbout-client/pom.xml [hunk 1]
```diff
@@ -101,13 +101,13 @@
...
             <artifactId>jersey-client</artifactId>
-            <version>1.18.1</version>
+            <version>1.19</version>
             <scope>runtime</scope>
         </dependency>
         <dependency>
             <groupId>com.jcabi</groupId>
             <artifactId>jcabi-http</artifactId>
-            <version>1.10.4</version>
+            <version>1.14.1</version>
```
Comment (dmzaytsev) [inline netbout-client/pom.xml]: @exper0 let's upgrade to latest 
https://github.com/jcabi/jcabi-http/releases/tag/1.14.2

**Comment by exper0** [comment]
@dmzaytsev done

**Comment by karato** [comment]
@exper0 Let me find a reviewer for this pull request, thanks for submitting it

**Comment by karato** [comment]
@pinaf review it

**Comment by pinaf** [comment]
@rultor merge

**Comment by dmzaytsev** [comment]
@rultor  merge

**Comment by karato** [comment]
@elenavolokhova please, let us know what do you think about this ticket, according to our [QA rules](http://at.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor deploy now pls

**Comment by elenavolokhova** [comment]
@pinaf 
According to our [quality rules](http://www.teamed.io/qa.html):
 > The code reviewer found at least three problems in the code.
 
 There were no issues found during review.
Please confirm that you will try to find at least three problems next time.

**Comment by pinaf** [comment]
@elenavolokhova I confirm

**Comment by elenavolokhova** [comment]
@karato Quality is acceptable here.

**Comment by karato** [comment]
> @karato Quality is acceptable here.

@elenavolokhova thanks, got it. everybody, please try to make it good next time

**Comment by karato** [comment]
@pinaf paied 10 mins to @elenavolokhova for QA review (payment ID is `74363052`)

**17 mins** added to your account (payment number `AP-3YD049137P536072A`), many thanks for your contribution! 1 hour and 41 mins spent here.

you're getting extra minutes here (c=2)

+17 added to your rating, at the moment it is: [+5361](http://www.netbout.com/b/35128?open=rating)

**Pull request merged**